cmd/plugin.go

```diff
@@ -209,26 +223,50 @@ var (
 
 	enableCommand = &cobra.Command{
 		Use:     "enable",
-		Short:   "Enabled or Disable plugin",
-		Args:    cobra.ExactArgs(2), // TODO: Can I check real git repo?
-		Example: "vatz plugin enable <pluginName> <true/false>",
+		Short:   "Enable plugin",
+		Example: "vatz plugin enable --plugin <pluginName>",
 		PreRunE: func(cmd *cobra.Command, args []string) error {
 			_, err := config.InitConfig(configFile)
 			return err
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			pluginName := viper.GetString("enable_plugin")
 			pluginDir, err := config.GetConfig().Vatz.AbsoluteHomePath()
 			if err != nil {
 				return err
 			}
 
-			log.Debug().Str("module", "plugin").Msgf("enable installed plugin %s at %s", args[0], pluginDir)
+			log.Debug().Str("module", "plugin").Msgf("enable installed plugin %s at %s", pluginName, pluginDir)
 
-			// TODO: Handle already installed.
-			// TODO: Handle invalid repo name.
 			mgr := plugin.NewManager(pluginDir)
-			enableDisable := utils.ParseBool(args[1])
-			err = mgr.Update(args[0], enableDisable)
+			err = mgr.Update(pluginName, true)
+			if err != nil {
+				log.Error().Str("module", "plugin").Err(err)
+				return err
+			}
+			return nil
+		},
+	}
+
+	disableCommand = &cobra.Command{
+		Use:     "disable",
+		Short:   "Disable plugin",
+		Example: "vatz plugin disable --plugin <pluginName>",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			_, err := config.InitConfig(configFile)
+			return err
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			pluginName := viper.GetString("disable_plugin")
+			pluginDir, err := config.GetConfig().Vatz.AbsoluteHomePath()
+			if err != nil {
+				return err
+			}
+
+			log.Debug().Str("module", "plugin").Msgf("enable installed plugin %s at %s", pluginName, pluginDir)
```

Comment:
`disable installed plugin` seems to be correct log here.

Reply:
+1